libcrux-ml-kem/src/vector/traits.rs

```diff
@@ -85,6 +85,8 @@ pub fn decompress_1<T: Operations>(v: T) -> T {
 /// Internal vectors.
 ///
 /// Used in the unpacked API.
+#[cfg(feature = "unpacked")]
 pub trait VectorType: Operations {}
 
+#[cfg(feature = "unpacked")]
```

Comment:
These types aren't actually publicy accessible and don't get docs. They need to be public though to have them in the trait bound for the type (though that's only there because eurydice chokes on it, it really shouldn't be there).